libcontainer/system/linux.go

```diff
@@ -53,6 +53,14 @@ func Execv(cmd string, args []string, env []string) error {
 	return syscall.Exec(name, args, env)
 }
 
+func Prlimit(pid, resource int, limit syscall.Rlimit) error {
+	_, _, err := syscall.RawSyscall6(syscall.SYS_PRLIMIT64, uintptr(pid), uintptr(resource), uintptr(unsafe.Pointer(&limit)), uintptr(unsafe.Pointer(&limit)), 0, 0)
```

Comment:
Thanks!

Sent from my iPhone

> On Mar 25, 2016, at 6:08 PM, Marcos Nils <notifications@github.com> wrote:
> 
> In libcontainer/system/linux.go:
> 
> > @@ -53,6 +53,14 @@ func Execv(cmd string, args []string, env []string) error {
> >  	return syscall.Exec(name, args, env)
> >  }
> >  
> > +func Prlimit(pid, resource int, limit syscall.Rlimit) error {
> > +	_, _, err := syscall.RawSyscall6(syscall.SYS_PRLIMIT64, uintptr(pid), uintptr(resource), uintptr(unsafe.Pointer(&limit)), uintptr(unsafe.Pointer(&limit)), 0, 0)
> @mrunalp seems like golang 386 package uses the same call also for 386 architectures (https://golang.org/src/syscall/zsyscall_linux_386.go?h=prlimit#L766)
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>